.github/workflows/test_integration.yml

```diff
@@ -54,7 +54,11 @@ jobs:
         with:
           node-version: 14
       - name: Report source code metadata
-        run: npx @datadog/datadog-ci git-metadata upload
+        run: |
+          mkdir -p $GITHUB_WORKSPACE/bin
+          curl -L "https://github.com/DataDog/datadog-ci/releases/latest/download/datadog-ci" --output "$GITHUB_WORKSPACE/bin/datadog-ci"
+          chmod +x "$GITHUB_WORKSPACE/bin/datadog-ci"
+          $GITHUB_WORKSPACE/bin/datadog-ci git-metadata upload
```

Comment:
@juan-fernandez do we need to build the package before uploading git metadata?

Reply:
what package? `datadog-ci`?